distributed/worker.py

```diff
@@ -3111,6 +3123,14 @@ def get_client(address=None, timeout=3, resolve_address=True):
     worker_client
     secede
     """
+
+    # Get timeout value from dash config if not passed.
+    # Defaults to 3s if the config is not set.
```

Comment:
I think that we can probably drop these comments.  I think that the code here is clear enough.

Reply:
Ack! Done.